Xamarin.Forms.Core/BindingExpression.cs

```diff
@@ -679,7 +675,7 @@ public void Subscribe(INotifyPropertyChanged handler)
 				// Clear out the old subscription if necessary
 				Unsubscribe();
 
-				_listener = new WeakPropertyChangedProxy(handler, _changeHandler);
+				_listener = new WeakPropertyChangedProxy(handler, PropertyChanged);
```

Comment:
Removing `_changeHandler` is actually breaking something, looking into it.

Reply:
Wait what, I wrote this??? 🤦‍♂ 

https://github.com/xamarin/Xamarin.Forms/blame/master/Xamarin.Forms.Core/BindingExpression.cs#L659